tests/loadtests/glloadtests.cmake

```diff
@@ -171,12 +171,14 @@ function( create_gl_target target version sources common_resources test_images
             GL_CONTEXT_PROFILE=${KTX_GL_CONTEXT_PROFILE}
             GL_CONTEXT_MAJOR_VERSION=${KTX_GL_CONTEXT_MAJOR_VERSION}
             GL_CONTEXT_MINOR_VERSION=${KTX_GL_CONTEXT_MINOR_VERSION}
+            $<$<PLATFORM_ID:Windows>:NOMINMAX>
```

Comment:
Was there no `#define NOMINMAX` in one of more the source files? If not how was this building before?

Reply:
I have no idea. There was one, but that did not prove sufficient. I don't think this issue is important enough to warrant an investigation and downgrade compilers, binary search CMake option combinations to track down how it's been working thus far. (See my previous comment, no code since the existence of the STL should want to rely on the `min` macro, so just turn it off everywhere.)

Reply:
See my comment in the related conversation. No compiler downgrade should be necessary.

I'm going to mark this conversation resolved.